nimbus-features/onboardingFrameworkFeature.yaml

```diff
@@ -30,7 +30,7 @@ features:
             ALWAYS:                   "true"
             NEVER:                    "false"
           cards:
-            - name: "welcome"
+            wecome:
               order: 10
```

Comment:
Not related to the PR but I'm curious why the values for order are in the tens?

Reply:
Ah. I did it so, in case we want to add cards in between two other existing cards, we could do that whit experiments more easily, rather than going in and changing each order manually.